sources/packages/forms/src/form-definitions/sfaa2025-26-pt.json

```diff
@@ -5188,29 +5189,22 @@
                 {
                   "label": "My total current year gross income is:",
                   "prefix": "$",
-                  "customClass": "font-weight-bold",
+                  "applyMaskOn": "change",
                   "mask": false,
-                  "spellcheck": true,
                   "tableView": false,
-                  "currency": "CAD",
+                  "delimiter": true,
+                  "requireDecimal": false,
                   "inputFormat": "plain",
+                  "truncateMultipleSpaces": false,
                   "validate": {
                     "required": true,
                     "custom": "valid = (input >= 0 && input < data.maxIncome) ? true : 'The number you have entered is outside the expected range.';"
                   },
+                  "validateWhenHidden": false,
                   "key": "currentYearIncome",
-                  "conditional": {
-                    "show": "true",
-                    "when": "hasSignificantDegreeOfIncome",
-                    "eq": "yes"
-                  },
```

Comment:
Please retain the condition in the number component.

Reply:
Updated - good catch.